app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -303,6 +303,27 @@ class ProfileEditActivityTest {
     }
   }
 
+  @Test
+  fun testProfileEdit_startWithUserProfile_clickDelete_configChange_checkDeletionDialogIsVisible() {
+    launch<ProfileEditActivity>(
+      ProfileEditActivity.createProfileEditActivity(
+        context,
+        profileId = 1
+      )
+    ).use {
+      onView(withId(R.id.profile_delete_button)).perform(click())
```

Comment:
Test may sometimes fail on the small screen devices if we don't perform `scrollTo()` before `click()`

Reply:
Done. Updated the test screenshot as well